contracts/connectors/loantoken/modules/beaconLogicWRBTC/LoanTokenMintAndBurnWrbtc.sol

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2017-2020, bZeroX, LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "../../LoanTokenLogicStandard.sol";
+
+contract LoanTokenMintAndBurnWrbtc is LoanTokenLogicStandard {
```

Comment:
Why are you here not also inheriting from LoanTokenLogicSplit ?

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/515/commits/7fee23c33c9468d4e032cd3d628da2a5a61a1138)